micrometer-core/src/main/java/io/micrometer/core/instrument/binder/tomcat/TomcatMetrics.java

```diff
@@ -147,6 +147,12 @@ private void registerThreadPoolMetrics(MeterRegistry registry) {
                     .tags(allTags)
                     .baseUnit(BaseUnits.THREADS)
                     .register(registry);
+
+            Gauge.builder("tomcat.threads.active", mBeanServer,
```

Comment:
Is the connectionCount necessarily equal to the number of active threads in the thread pool? Starting an application locally and looking at the MBean, I see `maxThreads` is 200, but `maxConnections` is 10,000.

Reply:
I set baseUnit to "connections", so whole metric is "tomcat_threads_active_connections".
Maybe you have a suggestion for a better name?